models/user_heatmap.go

```diff
@@ -32,9 +32,20 @@ func GetUserHeatmapDataByUser(user *User) ([]*UserHeatmapData, error) {
 		groupByName = groupBy
 	}
 
+	var isOrganization string
+	switch user.Type {
+	case UserTypeIndividual:
+		// For invidividual users only their own contributions count
+		isOrganization = "0"
+	case UserTypeOrganization:
+		// For organisations contributions by every user in owned repos count
+		isOrganization = "1" // mssql does not support boolean literal for expressions
+	}
+
 	err := x.Select(groupBy+" AS timestamp, count(user_id) as contributions").
```

Comment:
I would prefer this to be changed this way:

```go
sess := x.Select(groupBy+" AS timestamp, count(user_id) as contributions").
		Table("action").
		Where("user_id = ?", user.ID).
		And("created_unix > ?", (util.TimeStampNow() - 31536000))
if user.Type == UserTypeIndividual {
	sess = sess.And("act_user_id = ?", user.ID)
}
err := sess.GroupBy(groupByName).
		OrderBy("timestamp").
		Find(&hdata)
```

Reply:
Looks a lot simpler, I will add some comments and commit it.